### PR TITLE
update blog contract & add functional front-end

### PR DESCRIPTION
**Pull request by windushka**
This is a first version. Would gladly appreciate suggestions and remarks

### CODE REVIEW

File: blog/front/public/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+        <title>Massa - Sum example</title>
```
Comment (julienbrs) [inline blog/front/public/index.html:27]:
```suggestion
        <title>Massa - Blog example</title>
```

**Review by julienbrs — COMMENTED** [review]
Could you try to display an error message when there is no provider available and therefore not display the rest of the body page?

**Comment by windushka** [comment]
> Could you try to display an error message when there is no provider available and therefore not display the rest of the body page?

Good point. The loader component will be rendered before displaying the page or the error message if no providers were found.

**Review by julienbrs — APPROVED** [review]
Looks great!

**Review by Ben-Rey — APPROVED** [review]

**Pull request merged**